website/src/index.css

```diff
@@ -0,0 +1,13 @@
+/* body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
```

Comment:
will this be reactivated at a later point in time or can it be removed?

Reply:
removed